tests/components/Dropdown.test.tsx

```diff
@@ -168,7 +219,7 @@ describe('Dropdown', () => {
     expect(itemNode.className).not.toContain('FocusedItem1');
   });
 
-  it('Handles submission when pressing "Enter" in the input box', () => {
+  it('submits when pressing "Enter" in the input box', () => {
```

Comment:
"enter is pressed" and "when pressing Enter" is a little weird wording I'd probably just say "on Enter"

Reply:
changed